src/Sfa.Tl.ResultsAndCertification.Api.Client/Clients/DfeSignInApiClient.cs

```diff
@@ -32,34 +35,37 @@ public DfeSignInApiClient(HttpClient httpClient, ITokenServiceClient tokenServic
 
         public async Task<DfeUserInfo> GetDfeSignInUserInfo(string organisationId, string userId)
         {
-            var organisationUkprn = GetOrganisationUkprn(organisationId, userId);
+            var organisation = GetOrganisation(organisationId, userId);
             var userInfo = GetUserInfo(organisationId, userId);
 
-            await Task.WhenAll(organisationUkprn, userInfo);
+            await Task.WhenAll(organisation, userInfo);
 
             var userInfoResult = userInfo.Result;
-            var ukprn = organisationUkprn.Result;
+            var ukprn = organisation.Result;
+            userInfoResult.Organisation = organisation.Result.Name;
```

Comment:
The organisation name is only copied to userInfoResult.Organisation to be compared later inside the HasAccesstoService method. Why don't just compare the value using organisation.Result.Name? And remove the Organisation property in the DfeUserInfo class?

Reply:
the ukprn variable should be named organisation

Reply:
that is the change.